distribution/test/itests/pom.xml

```diff
@@ -125,6 +117,61 @@
             <artifactId>javax.inject</artifactId>
         </dependency>
         <!-- End of pax exam dependencies -->
+        <dependency>
```

Comment:
@pklinef I was able to push some stuff down. We'll see what the hero build does.